jsapp/js/components/newFeatureDialog.module.scss

```diff
@@ -0,0 +1,49 @@
+@use '~kobo-common/src/styles/colors';
+
+.root {
+	position: relative;
+  z-index: 1100; // 1 index lower than a modal
+}
+
+.wrapper {
+	position: relative;
+}
+
+.dialog {
+  width: 50%;
+  position: absolute;
+  border-radius: 4px;
+  padding: 12px;
+  color: colors.$kobo-white;
+  background: colors.$kobo-dark-blue;
+}
+
+.dialog::before {
+  content: " ";
+  left: 5%;
+  border: solid transparent;
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+  border-width: 6px;
```

Comment:
The pixel dimensions in this file should probably use the values from `scss/sizes`.